packages/mdc-checkbox/index.js

```diff
@@ -30,6 +32,34 @@ export class MDCCheckbox extends MDCComponent {
     return this.root_.querySelector(NATIVE_CONTROL_SELECTOR);
   }
 
+  constructor(...args) {
+    super(...args);
+    this.ripple_ = this.initRipple_();
+  }
+
+  initRipple_() {
+    const adapter = Object.assign(MDCRipple.createAdapter(this), {
+      isUnbounded: () => true,
+      isSurfaceActive: () => false,
```

Comment:
Good call. The checkbox behaves as expected when tabbing through and pressing space and only space to actuate the checkbox. This can be removed.